src/components/Banner.tsx

```diff
@@ -1,25 +1,28 @@
 import React from 'react'
-import { Button } from 'reactstrap';
+import Button from './ui/button/Button'
+import { Button as Btn} from 'reactstrap';
```

Comment:
Use One library for the component, either use custom button or reactstrap. please DO NOT use two library for the same component.

Reply:
done